docs/docs/troubleshooting.md

```diff
@@ -164,3 +164,16 @@ contrast version v0.X.0
     image versions:       ghcr.io/edgelesssys/contrast/coordinator@sha256:...
                           ghcr.io/edgelesssys/contrast/initializer@sha256:...
 ```
+
+## VM runs out of memory
+
+Since pod VMs are statically sized, it's easier to run out of memory due to misconfigurations.
+Setting the right memory limits is even more important on bare metal, where the image layers need to be stored in the guest memory, too.
+If the guest pull fails due to lack of memory, you might see an error message like this:
```

Comment:
Would turn that sentence around: if you see the error, it might be failing cos of memory.

Reply:
Done